Library/Homebrew/rubocops/shared/desc_helper.rb

```diff
@@ -53,7 +53,7 @@ def audit_desc(type, name, desc_call)
         desc_problem "Description shouldn't start with an article." if regex_match_group(desc, /^(the|an?)(?=\s)/i)
 
         # Check if invalid lowercase words are at the start of a desc.
-        if !VALID_LOWERCASE_WORDS.include?(string_content(desc).split.first) &&
+        if !VALID_LOWERCASE_WORDS.include?(string_content(desc).split.first) && # rubocop:disable Style/InverseMethods (false positive)
```

Comment:
This triggered errors when i applied the autocorrect, so I've disabled the cop instead